dockerfiles/ubuntu/obkm/Dockerfile

```diff
@@ -67,15 +68,17 @@ RUN \
         unzip \
         wget \
     && rm -rf /var/lib/apt/lists/*
+
 # add the WSO2 product distribution to user's home directory
 RUN \
     wget --no-check-certificate -O ${WSO2_SERVER}.zip "${WSO2_SERVER_DIST_URL}" \
     && unzip -d ${USER_HOME} ${WSO2_SERVER}.zip \
     && chown wso2carbon:wso2 -R ${WSO2_SERVER_HOME} \
     && rm -f ${WSO2_SERVER}.zip
+
 # add libraries for Kubernetes membership scheme based clustering
-ADD --chown=wso2carbon:wso2 https://repo1.maven.org/maven2/dnsjava/dnsjava/${DNS_JAVA_VERSION}/dnsjava-${DNS_JAVA_VERSION}.jar ${WSO2_SERVER_HOME}/repository/components/lib/
-ADD --chown=wso2carbon:wso2 https://repo1.maven.org/maven2/org/wso2/carbon/kubernetes/artifacts/kubernetes-membership-scheme/${K8S_MEMBERSHIP_SCHEME_VERSION}/kubernetes-membership-scheme-${K8S_MEMBERSHIP_SCHEME_VERSION}.jar ${WSO2_SERVER_HOME}/repository/components/dropins/
+ADD --chown=wso2carbon:wso2 https://repo1.maven.org/maven2/dnsjava/dnsjava/${DNS_JAVA_VERSION}/dnsjava-${DNS_JAVA_VERSION}.jar ${WSO2_SERVER_HOME}/repository/components/lib
+ADD --chown=wso2carbon:wso2 http://maven.wso2.org/nexus/content/repositories/releases/org/wso2/carbon/kubernetes/artifacts/kubernetes-membership-scheme/${K8S_MEMBERSHIP_SCHEME_VERSION}/kubernetes-membership-scheme-${K8S_MEMBERSHIP_SCHEME_VERSION}.jar ${WSO2_SERVER_HOME}/repository/components/dropins
```

Comment:
Aren't we decided to avoid packaging these libraries with the Docker image?. Instead of plugging those at the container orchestration layer.

Reply:
@ThilinaManamgoda this is the case with external libraries and extensions such as, the JDBC drivers. For example, a customer would want to integrate a different version of MySQL JDBC driver compared to the default version packaged by WSO2 thus, leading to conflicts.

This is not the case with a WSO2 specific implementation such as, the membership schemes where we always recommend using the most latest version in order to obtain latest improvements and bug fixes.

Reply:
@chirangaalwis This assumption is true only if the given docker images are only targeted for K8s, isn't it?

Reply:
Yes, as of now. But if we use installation resources for a different container orchestration platform, we may need to do the same.

In fact @ThilinaManamgoda, yes I agree that the design you suggest for the membership scheme packaging is cleaner than packaging it in the default container images. As per your suggestion, we can include this only during a specific platform deployment. But since, container images for the product versions for which we currently package these binaries are already released, even if we adopt the suggested strategy we may have to do it in installation resources for a fresh version.

Reply:
@chirangaalwis Got it. To avoid breaking the existing deployments, let's move forward with packaging the K8s membership schema.